share/completions/wg-quick.fish

```diff
@@ -0,0 +1,18 @@
+set -l valid_subcmds up down strip save
+
+function __wg_complete_interfaces
+    wg show 2>| string replace -rf 'Unable to access interface (.*): Operation not permitted' '$1'
+end
+
+function __wg_subcmd --no-scope-shadowing
+    set -l subcmd $argv[1]
+    set -e argv[1]
+    complete -f -c wg-quick -n "not __fish_seen_subcommand_from $valid_subcmds" -a $subcmd $argv
```

Comment:
Please just repeat this further down.

This is a fairly awkward abstraction, removing it makes things more normal and simpler.